privacy/privacy-shield.md

```diff
@@ -58,18 +58,18 @@ In order to improve our services and the website, and provide more convenient, r
 ## Third Parties
 You understand that Basecamp uses third party vendors and hosting partners to provide the necessary hardware, software, networking, storage, and related technology required to provide you with our services. A current list of vendors is available upon request.
 
-In cases of onward transfer to these third parties for data of EU or Swiss individuals received pursuant to the Privacy Shield Frameworks, Basecamp is potentially liable should any issues or concerns arise.
+In cases of onward transfer to these third parties for data of EU, United Kingdom or Swiss individuals received pursuant to the Privacy Shield Frameworks, Basecamp is potentially liable should any issues or concerns arise.
 
 ## Deleted Data
 When you cancel your account, we'll ensure that nothing is stored on our servers past 30 days. Anything you delete on your account while it's active will also be purged within 30 days (up until then it's available in the trash can).
 
 ## EU-US and Swiss-US Privacy Shield Framework
-Basecamp complies with the EU-U.S. Privacy Shield Framework and the Swiss-U.S. Privacy Shield Framework as set forth by the U.S. Department of Commerce regarding the collection, use, and retention of personal information transferred from the European Union and Switzerland to the United States, respectively. Basecamp has certified to the Department of Commerce that it adheres to the Privacy Shield Principles. If there is any conflict between the terms in this privacy policy and the Privacy Shield Principles, the Privacy Shield Principles shall govern. To learn more about the Privacy Shield program, and to view our certification, please visit <a href="https://www.privacyshield.gov/">https://www.privacyshield.gov/</a>.
+Basecamp complies with the EU-U.S. Privacy Shield Framework and the Swiss-U.S. Privacy Shield Framework as set forth by the U.S. Department of Commerce regarding the collection, use, and retention of personal information transferred from the European Union and the United Kingdom and Switzerland to the United States, respectively. Basecamp has certified to the Department of Commerce that it adheres to the Privacy Shield Principles. If there is any conflict between the terms in this privacy policy and the Privacy Shield Principles, the Privacy Shield Principles shall govern. To learn more about the Privacy Shield program, and to view our certification, please visit <a href="https://www.privacyshield.gov/">https://www.privacyshield.gov/</a>.
```

Comment:
"A and B and C" reads a bit awkwardly. Go with commas?

Reply:
Mmm, yes. The model language uses the repeated and, but this seems like a place where serial commas would work nicely.

```suggestion
Basecamp complies with the EU-U.S. Privacy Shield Framework and the Swiss-U.S. Privacy Shield Framework as set forth by the U.S. Department of Commerce regarding the collection, use, and retention of personal information transferred from the European Union, the United Kingdom, and Switzerland to the United States, respectively. Basecamp has certified to the Department of Commerce that it adheres to the Privacy Shield Principles. If there is any conflict between the terms in this privacy policy and the Privacy Shield Principles, the Privacy Shield Principles shall govern. To learn more about the Privacy Shield program, and to view our certification, please visit <a href="https://www.privacyshield.gov/">https://www.privacyshield.gov/</a>.
```